src/main/java/androidx/graphics/Bitmap.kt

```diff
@@ -111,3 +119,49 @@ inline fun createBitmap(
 ): Bitmap {
     return Bitmap.createBitmap(width, height, config, hasAlpha, colorSpace)
 }
+
+
+/**
+ * Bitmap to bytes.
+ *
+ * @param format The format of bitmap.
+ * @param quality Hint to the compressor, 0-100. 0 meaning compress for small size, 100 meaning compress for max quality.
+ * @return ByteArray
+ */
+fun Bitmap.toBytes(format:CompressFormat = JPEG, quality: Int=100):ByteArray
```

Comment:
Compressing a bitmap can be done in many ways and compressing into a byte array is just a highly specific use case. I'd rather remove this method.